src/main/java/com/linkedin/kmf/apps/MirrorPipelineMonitor.java

```diff
@@ -70,6 +70,7 @@ public MirrorPipelineMonitor(Map<String, Object> props, String name) throws Exce
                                                                      String topic,
                                                                      String name) throws Exception {
     Map<String, Object> topicManagementProps = createTopicManagementServiceProps(props, topic);
+    topicManagementProps.put(CommonServiceConfig.ZOOKEEPER_CONNECT_CONFIG, "");
```

Comment:
@WarrenGreen The reviewer usually makes suggestion based on the code snippet and it is possible that the comment itself may be wrong. You probably need to address the comment that you think is right, and defend the approach if you think the comment is wrong, since you own the code and have spent more time on the patch than anyone else.

I am not sure this can be a TODO. I think the goal is for us to monitor a pipeline of multiple clusters even when the number of live brokers can change, right?